torch/csrc/jit/runtime/register_prim_ops_fulljit.cpp

```diff
@@ -127,6 +128,60 @@ RegisterOperators reg(
            };
          },
          aliasAnalysisSpecialCase()),
+     Operator(
+         "prim::AllocateSlab() -> Storage",
+         [](const Node* node) -> Operation {
+           int64_t total_size = node->i(attr::total_size);
+           auto device_type =
+               static_cast<DeviceType>(node->i(attr::device_type));
+           return [total_size, device_type](Stack* stack) {
+             auto allocator = GetAllocator(device_type);
+             auto slab = c10::Storage(
+                 c10::Storage::use_byte_size_t(),
+                 total_size,
+                 allocator,
+                 /*resizable=*/false);
+             push(stack, std::move(slab));
+           };
+         },
+         aliasAnalysisSpecialCase()),
```

Comment:
why does this require aliasAnalysisSpecialCase ?

Reply:
actually i'm not sure about this one - another hold over from when i first started before i knew up from down. I figured someone would correct me here

Reply:
yea, you can just use AliasAnalysisFromSchema I think

Reply:
hmm I get this when making that substitution

```
terminate called after throwing an instance of 'c10::Error'
  what():  The operator prim::PreallocateTensor is special cased and cannot use explicit alias analysis.
```